### DISCUSSION TITLE
Is the "Static Exports" feature still available in the App Router?

### DISCUSSION THREAD

**Discussion by Yawanseh** [vercel/next.js#51624] [App Router] [answered]
Hello 👋,

I am particularly interested in knowing if "Static Exports" is still supported in the current version of the Next.js with App Router. If it is, could you please provide some details or examples on how it works? If it's not, what are the alternatives for achieving similar functionality?

Thanks in advance for your help!

**Comment by joaopedrodcf** [comment, marked as answer]
The feature is still available as you can check here on the docs page: https://nextjs.org/docs/app/building-your-application/deploying/static-exports

Also as you can see it also supports server components 👌
